runtime/vm/JFRChunkWriter.hpp

```diff
@@ -426,6 +422,8 @@ class VM_JFRChunkWriter {
 
 			if (dumpCalled) {
 				writeThreadDumpEvent();
+				pool_do(_constantPoolTypes.getSystemProcessTable(), &writeSystemProcessEvent, this);
+				pool_do(_constantPoolTypes.getNativeLibraryTable(), &writeNativeLibraryEvent, this);
```

Comment:
We should arrange to avoid the expense of filling the process and native library tables if they're not going to be usd.

Reply:
@thallium please make that change